### DISCUSSION TITLE
Add color variants for Gruvbox theme

### DISCUSSION THREAD

**Discussion by OBLIXi-ON** [NvChad/NvChad#2954] [Ideas]
There are 3 color variants for Gruvbox theme(i.e. hard, medium and soft). The one that NvChad uses is soft. Add the remaining ones too please :crying_cat_face:.

The one that VsCode uses is the `original` one:
https://github.com/sainnhe/gruvbox-material?tab=readme-ov-file

**Comment by siduck** [comment]
@OBLIXi-ON nvchad has gruvchad which is material , i dont see much diff  between hard medium soft, u can change bg color of normal hl group too btw

**Reply by OBLIXi-ON** [reply]
Yeah I mean only the background is a bit darker. Also since you've gruvchad which is a custom made theme, probably I can make my own themes too. Any instructions on how can I make and implement one?

Also thanks

**Reply by siduck** [reply]
@OBLIXi-ON  so you just want to change the background?

**Reply by OBLIXi-ON** [reply]
Yeah for now. I didn't actually check the foreground color codes, they all seemed the same except the background which is a bit contrasty (`hard`>`medium`>`soft`).

Also `Gruvchad` inspired me to make my own themes 😅

**Reply by siduck** [reply]
ok so try this 
```lua
M.base46 =   {

hl_override =   {
    Normal = { bg = "#your hexcolor" }
}

 }
